perun-wui-consolidator/src/main/java/cz/metacentrum/perun/wui/consolidator/widgets/Wayf.java

```diff
@@ -128,7 +128,7 @@ public void onClick(ClickEvent clickEvent) {
 							authnContextClassRef += "%20urn:cesnet:proxyidp:filter:"+group.getFilter();
 						}
 
-						final String redirectUrl = PerunConfiguration.getWayfSpLogoutUrl() + "?return=" + PerunConfiguration.getWayfSpLoginUrl() + URL.encodeQueryString("?authnContextClassRef=" + authnContextClassRef +"&target="+consolidatorUrl);
+						final String redirectUrl = PerunConfiguration.getWayfSpLogoutUrl() + "?return=" + PerunConfiguration.getWayfSpLoginUrl() + URL.encodeQueryString("?authnContextClassRef=" + authnContextClassRef +"&target="+consolidatorUrl + "&forceAuthn=true");
 						Window.Location.assign(redirectUrl);
```

Comment:
This works for ``fed`` type of wayf button we use at CESNET, but what about ``direct`` (few lines below). It is expected use-case for instances with single proxy (elixir, bbmri,...), where consolidator offers only "continue" like button and force authn is expected to be set on SP/Proxy, since it knows, it must force authn for consolidator SP.
I believe, you want this case to be updated too, but you probably must modify url to contain also explicit login, since it is the one respecting forceAuthn param.